src/onegov/event/collections/occurrences.py

```diff
@@ -303,6 +389,24 @@ def query(self):
         if self.tags:
             query = query.filter(Occurrence._tags.has_any(array(self.tags)))
 
+        if self.filter_keywords:
+            keywords = self.valid_keywords(self.filter_keywords)
+
+            values = [val for sublist in keywords.values() for val in sublist]
+            values.sort()
+
+            values = [
+                Event.filter_keywords.contains(value)
```

Comment:
@Daverball I could not figure out how to properly filter for the selected keywords on the UI. Any idea?
(code runs but filtering is not working)

Reply:
@Tschuppi81 That's because `dict_property` by default assume a string value, I will probably get rid of that, I had a good idea how to infer the value type from the annotation on the left hand side.

But in the meantime you can do `content_property(value_type=list)` to force it to not coerce to a string in SQL expressions, then `contains` should work just fine.

Reply:
Thanks. I did set the value type to dict as I stored keywords in this form `{"filter_keywords": {"kalender": "DWS Sport", "sportart": "Fussball"}}`. I guesss I need to get the values first and then need to test...